var/spack/repos/builtin/packages/vim/package.py

```diff
@@ -124,6 +127,17 @@ def configure_args(self):
 
         return configure_args
 
+    @run_before('configure')
+    def select_features(self):
+        """ Modify the build by editing a header file containing
+            defines for optional code and preferences. """
+        featurefile = FileFilter("src/feature.h")
+        sys_vimrc_file = self.spec.variants['sys_vimrc_file'].value
+        if sys_vimrc_file != 'none':
+            featurefile.filter('/\* #define SYS_VIMRC_FILE	"/etc/vimrc" \*/',
```

Comment:
I'd suggest using raw strings (`r'content'`) for regex. That should help with your flake errors.